msupply-foundation/unified-codes
# Logo only navigates once

Fixes #447

## Description


Changes logo to be a link component so that it only navigates the user if we aren't already on that route. 

To test:
- While unauthenticated, navigate to a details page
- Click on logo to return to browse page
- Click the logo a bunch more times
- Click the back arrow in your browser 
- Result: You should only need to click the back arrow once to return to the details page
